src/pdf/proofPage.js

```diff
@@ -119,8 +117,8 @@ function structNlProof(doc, qrSvg, proof, createdAt, keyId) {
         doc.pdf.struct("Sect", proofContent),
     ];
 
-    if (createdAt || keyId) {
-        content.push(structNlFooter(doc, createdAt, keyId));
+    if (proof.keyIdentifier && !proof.validAtMost25Hours) {
+        content.push(structNlFooter(doc));
```

Comment:
I'm really not sure, but isn't the footer always visible if the `keyIdentifier` is present, regardless of whether it's considered a test or not? I thought the check for negative test was only for explanation about validities.

Reply:
That is correct. If I understood correctly, the blue bar should always be there when `keyIdentifier` is present, and the text should be hidden for negativetest certs. The name `structNlFooter` doesn't communicate this clearly enough, it's only for the "structured content" part of the footer, since the blue background is a non-content artifact and therefore outside of the content structure.

Although, on second thought... Unlike e.g. the back for the "questions?" info box, it does have meaning. I'll change it to a tagged Illustration.

Reply:
I'll park this for now.